src/openforms/js/components/admin/forms/FormRow.stories.mdx

```diff
@@ -19,14 +19,43 @@ export const Template = args => (
   </FormRow>
 );
 
-## Presentation
+## Single field
 
 <Canvas>
-  <Story name="Default" args={{}}>
+  <Story name="Single field" args={{}}>
     {Template.bind({})}
   </Story>
 </Canvas>
 
+## Multiple fields
+
+export const FormDecorator = Story => (
+  <form>
+    <Fieldset>
+      <Story />
+    </Fieldset>
+  </form>
+);
+
+<Canvas>
+  <Story name="Multiple fields" decorators={[FormDecorator]} args={{}}>
```

Comment:
I think the fact that the single field is not wrapped in the fieldset, while the multiple fields story is wrapped, makes it look a bit inconsistent?
![Screenshot from 2023-02-28 10-42-40](https://user-images.githubusercontent.com/19154114/221814918-79d64875-220a-4252-b2dd-253971584c9d.png)

Reply:
right - I added the fieldset wrapper to get the Django styling to apply, but completely looked over the misaligned single field errors! I'll add the decorator too.